deploy/preprod/cron-early-allocation-events.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: early-allocation-events
+spec:
+  schedule: "00 23 * * *"
```

Comment:
It would be preferable to run this once a day _in the early morning_ rather than _at the end of the day_ – e.g. `00 02 * * *` rather than `00 23 * * *`.

This will help us catch when someone stops being eligible due to their sentence date having passed. Otherwise we'll effectively be > 23 hours late to emit the event.